pkg/northbound/gnmi/set.go

```diff
@@ -169,7 +172,12 @@ func (s *Server) buildUpdateResults(targetUpdates mapTargetUpdates,
 		} else if err == nil && cont {
 			continue
 		}
-
+		success, err := listenForDeviceResponse(networkChanges, target, configName)
```

Comment:
this blocks until it gets the response?

Reply:
yes it does. Probably ought to have a timeout. that's why I had the callback TODO in there.